testing/run_tests.sh

```diff
@@ -0,0 +1,10 @@
+#!bin/bash
+
+set -e
+
+echo "Expect to run from docker, in callek/shipit-test-runner"
+echo "  docker run --rm -v `pwd`:/src -ti callek/shipit-test-runner /src/testing/run_tests.sh"
```

Comment:
this script is for running tests, and it calls tox for us, and it does the npm install/run command.  Its mainly a helper so you don't have to type/copy a really long command line. But I'm open to other ideas to achieve the same result (I just don't like the idea of needing to use `npm install` in tox especially since `npm install` ends up requiring phantomjs which needs to be installed to work...